dashboard/security.py

```diff
@@ -36,18 +36,15 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def security(admin_only=False):
-    def decorator(f):
-        print(admin_only)
-        return f
-    return decorator
+def is_admin():
+    return session and session.pop(config.link + '_admin', False)
```

Comment:
looking only at these two lines, it seems to me that you would use `peek` or `top` instead of `pop` which modifies the stack. or?

Reply:
You're completely right, this was a bug; it should be fixed now.